### PR TITLE
activities marshmallow schema change

### LINKED ISSUE #6467: Deduplicate marshmallow schema by using marshmallow-sqlalchemy
There is a lot of repetition between sqlalchemy models and marshmallow schema

Use https://marshmallow-sqlalchemy.readthedocs.io to remove this duplication

This is a parent issue and new contributors can take an API component and send contributions converting their schemas

### PR DESCRIPTION
**Pull request by DiptoChakrabarty**
Fixes #6467 

#### Short description of what this resolves:
Uses the marshmallow-sqlalchemy  schema for the activities schema

#### Changes proposed in this pull request:

- Replaced the activities schema
- Used the latest marshmallow-sqlalchemy schema for replacement 
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Which issue is it fixing?

**Comment by DiptoChakrabarty** [comment]
> Which issue is it fixing?

I mentioned the issue

File: app/api/schema/activities.py [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
         inflect = dasherize
 
-    id = fields.Str(dump_only=True)
-    actor = fields.Str(allow_none=True)
-    time = fields.DateTime(allow_none=True)
-    action = fields.Str(allow_none=True)
+        model = Activity
+        dump_only = ("id",)
+        load_instance = True
+        include_fk = True
```
Comment (iamareebjamal) [inline app/api/schema/activities.py:22]: All of this needs to go in meta class, not here

Comment (DiptoChakrabarty) [reply app/api/schema/activities.py]: I have been trying to check the issue , as per the latest documentation of marshmallow sqlalchemy all this was put under the meta class only

Comment (iamareebjamal) [reply app/api/schema/activities.py]: That is, indeed, what I said

**Pull request closed without merging**